src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/CmdLineClient.kt

```diff
@@ -39,6 +40,11 @@ class PublisherArgs(parser: ArgParser) {
             "-t", "--publish-topic",
             help = "topic where zip will be published"
     )
+
+    val resultTopic by parser.storing(
+            "--result-sub",
```

Comment:
The variable is called "topic".
The argument and its help use the word "subscription"

Please remove this uncertainty.

Reply:
Done